getimage.py

```diff
@@ -0,0 +1,48 @@
+import httplib, urllib, base64, random, string, json, pprint, cStringIO
+from PIL import Image
+import io 
+
+def get_random_image(): 
+    search_term = "".join(random.choice(string.ascii_lowercase) for j in range(random.randint(1,6)))
+
+    headers = {
+        'Ocp-Apim-Subscription-Key': '9eca18ad3a9f4d16990d07cf0b86221b',
+    }
+
+    params = urllib.urlencode({
+        'q': search_term
+    })
+    try:
+        conn = httplib.HTTPSConnection('api.cognitive.microsoft.com')
```

Comment:
Have you ever tried [requests](http://docs.python-requests.org/en/master/)?
You could simplify the POST request.